app/scripts/src/content/search.js

```diff
@@ -59,20 +59,24 @@ Search.prototype = {
   findEpisode: function(options) {
     this.findItem({
       success: function(response) {
-        Request.send({
-          method: 'GET',
-          url: this.getEpisodeUrl(response['show']['ids']['slug']),
-          success: function(resp) {
-            if (this.item.episode) {
-              options.success.call(this, JSON.parse(resp));
-            } else {
-              this.findEpisodeByTitle(resp, options);
+        if (!response) {
```

Comment:
Yes it is needed. If response is empty it will cause a null-pointer at line 67 below (url). Since this is not part of the request, the error-callback will never be triggered, and the error will only be visible in the console, not in the log.